contracts/ClearingHouse.sol

```diff
@@ -382,7 +383,11 @@ contract ClearingHouse is
                 isExactInput: isBaseToQuote,
                 base: response.base,
                 quote: response.quote,
-                oppositeAmountBound: _getOppositeAmount(params.oppositeAmountBound, response.isPartialClose)
+                oppositeAmountBound: _getOppositeAmount(
+                    params.oppositeAmountBound,
+                    response.isPartialClose,
+                    response.closedRatio
+                )
```

Comment:
can we just remove this function and use:

```
isPartialClose ? oppositeAmountBound.mulRatio(response.closedRatio) : oppositeAmountBound;
```

Reply:
fixed in https://github.com/perpetual-protocol/perp-curie-contract/pull/90/commits/fb025ed686e167212354f8af67645b56edf9222d